CIP-0068/README.md

```diff
@@ -0,0 +1,223 @@
+
+To find the metadata for the `user token` you need to look for the output, where the `reference NFT` is locked in. How this is done concretely will become clear below. Moreover, this output contains a datum, which holds the metadata. The advantage of this approach is that the issuer of the assets can decide how the transaction output with the `reference NFT` is locked and further handled. If the issuer wants complete immutable metadata, the `reference NFT` can be locked at the address of an unspendable script. Similarly, if the issuer wants the NFTs/FTs to evolve or wants a mechanism to update the metadata, the `reference NFT` can be locked at the address of a script with arbitrary logic that the issuer decides.
+
+Lastly and most importantly, with this construction, the metadata can be used by a PlutusV2 script with the use of reference inputs [CIP-0031](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0031). This will drive further innovation in the token space. 
+
+## Specification
+
+This is the registered `asset_name_label` value
+
+| asset_name_label            | class        | description                                               |
+| --------------------------- | ------------ | --------------------------------------------------------- |
+| 100                         | NFT          | Reference NFT locked at a script containing the datum     |
+
+For a correct relationship between the `user token` and the `reference NFT` a few conditions **must** be met.
+- The `user token` and `reference NFT` **must** be under the same policy ID. 
+- For a specific `user token` there **must** exist exactly **one** `reference NFT`
+- The `user token` and associated `reference NFT` **must** follow the standard naming pattern. The asset name of both assets is prefixed with its respective `asset_name_label` followed by a pattern defined by the asset class (e.g. asset_name_label 222)
+
+Some remarks about the above,
+1. The `user token` and `reference NFT` do not need to be minted in the same transaction. The order of minting is also not important.
+2. It may be the case that there can be multiple `user tokens` referencing the same `reference NFT`. More explicit with this standard, if a specific `user token` has a quantity bigger than 1, then they all reference the same `reference NFT`.
+
+The datum in the output with the `reference NFT` contains the metadata at the first field of the constructor 0. The version number is at the second field of this constructor:
+```
+big_int = int / big_uint / big_nint
+big_uint = #6.2(bounded_bytes)
+big_nint = #6.3(bounded_bytes)
+
+metadata =
+  { * metadata => metadata }
+  / [ * metadata ]
+  / big_int
+  / bounded_bytes
+  
+version = big_int
```

Comment:
Are we anticipating more than `18446744073709551615` versions :grimacing: ?

Reply:
https://github.com/input-output-hk/cardano-ledger/blob/2eafa6e55723c4cd2ef04911674e8f66830661d5/eras/babbage/test-suite/cddl-files/babbage.cddl#L292

I was referring here to `plutus_data` and what it allows me to use. Better to use `bounded_bytes` or what else would you suggest?

Reply:
`int` ?